services/data-info/src/data_info/routes/data.clj

```diff
@@ -130,7 +130,7 @@ with characters in a runtime-configurable parameter. Currently, this parameter l
 
       (GET* "/chunks-tabular" [:as {uri :uri}]
         :query [params TabularChunkParams]
-        :return (s/either TabularChunkDoc TabularChunkReturn)
+        :return (s/doc-only TabularChunkReturn TabularChunkDoc)
```

Comment:
I switched the order here, because it seemed better to put the one that really is used first, and the documentation second.